site/docs/components/avatar.mdx

```diff
@@ -33,22 +34,26 @@ import WhenNotToUse from "docs-components/WhenNotToUse"
 
 ## To keep in mind
 
-- Where possible, link the avatar to details about the individual or group it represents:
-  - Where appropriate, link the avatar to a context-specific view of that individual or group. Otherwise, link to their canonical "profile" page.
+- Where possible, link the avatar to details about the individual, group or company it represents:
```

Comment:
Is this applicable for an AvatarGroup? You can't link an individual Avatar in a group, can you?

Reply:
Afaik this is mostly a copy from Avatar, isn't it? I wonder if we omit anything that is directly Avatar related and just link to it as a reference?

Reply:
I've made some minor tweaks to change this to just individual or company since the `group` term was a little confusing for a single avatar.